pkg/cache/cluster.go

```diff
@@ -1240,16 +1296,65 @@ func (c *clusterCache) processEvent(event watch.EventType, un *unstructured.Unst
 		return
 	}
 
+	c.eventMetaCh <- eventMeta{event, un}
+}
+
+func (c *clusterCache) processEvents() {
+	log := c.log.WithValues("fn", "processItems")
```

Comment:
Is "fn" a standard thing? I'd rather use a more verbose name.

Reply:
I completely agree with using a more verbose name. Fixed - https://github.com/argoproj/gitops-engine/commit/7a53ecab1d0eab6197dd11221f7c0b9ed3bed738